src/simulation/SimulationManager.java

```diff
@@ -18,18 +18,19 @@
 import java.io.File;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
```

Comment:
Is this import still required?

Reply:
yes, it is needed [here](https://github.com/WatForm/aldb/blob/master/src/simulation/SimulationManager.java#L506) for additionalsigscopes in the parsingConf